homefinder/backend/database_prepare.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3 as sq
+
+def sql_init():
+    # connect SQL server
+    conn = sq.connect('mysql.db')
+    # conn = sq.connect("file::memory:?cache=shared")
+
+    c = conn.cursor()    
+    c.executescript('''
+    drop table if exists property_table;
+    CREATE TABLE property_table (
```

Comment:
cur.execute('''
        CREATE TABLE property_table (
            property_id INTEGER PRIMARY KEY AUTOINCREMENT,
            property_name VARCHAR(255),
            description VARCHAR(255),
            num_bedroom INT,
            num_bathroom INT,
            area_size INT,
            price FLOAT(10,2),
            transaction_type TEXT CHECK( transaction_type IN ('Buy', 'Rent') ),
            property_type TEXT CHECK( property_type IN ('Condo', 'Apartment', 'House') ),
            parking TEXT CHECK( parking IN ('garage', 'underground', 'covered', 'outdoor') ),
            laundry TEXT CHECK( laundry IN ('in_suite', 'shared') ),
            furnished BOOL,
            pet_friendly BOOL,
            longitude decimal(10,7),
            latitude decimal(10,7),
            build_year YEAR,
            smoking_allowed Bool,
            air_conditioning Bool,
            hardwood_floors Bool,
            balcony Bool
        );
    ''')